example/kustomization.yaml

```diff
@@ -64,7 +64,7 @@ resources:
 # Katib
 - ../apps/katib/upstream/installs/katib-with-kubeflow
 # Central Dashboard
-- ../apps/centraldashboard/upstream/overlays/oauth2-proxy
+- ../apps/centraldashboard/manuel-patches/oauth2-proxy
```

Comment:
Probably we should call it just overlays instead of manuell-patches

Reply:
this is fine for me